src/spaceone/identity/manager/mfa_manager/base.py

```diff
@@ -108,12 +108,19 @@ def get_mfa_info(credentials: dict):
             return cached_mfa_info
         raise ERROR_INVALID_CREDENTIALS()
 
+    def _generate_verify_code(self):
+        otp_secret_key = self.generate_otp_secret_key()
+        otp = self.generate_otp(otp_secret_key)
+        verify_code = otp.now()
+
+        return verify_code
+
```

Comment:
How about this code?
```suggestion
import secrets

def generate_secure_numeric_code(length=6):
    first_digit = str(secrets.randbelow(9) + 1)
    remaining_digits = ''.join(str(secrets.randbelow(10)) for _ in range(length - 1))
    verification_code = first_digit + remaining_digits

```